specification/devopsinfrastructure/Microsoft.DevOpsInfrastructure.Management/client.tsp

```diff
@@ -7,3 +7,119 @@ using Azure.ClientGenerator.Core;
   "DevOpsInfrastructureMgmt",
   "python"
 );
+
+namespace Microsoft.DevOpsInfrastructure {
+  @@access(Operations.list, Access.internal);
+
+  /* Modify the usage to ensure backward compatibility with the SDK generated from Swagger. */
+  @@usage(ResourceSkuProperties, Usage.input | Usage.output, "csharp");
```

Comment:
since this service is not GA yet, shall we keep the correct usages for them?

Reply:
fixed